modules/ROOT/pages/changelogs.adoc

```diff
@@ -1,367 +1,27 @@
-:description: This page lists all changes to status codes per Neo4j version.
-= Changes to status codes per Neo4j version
+:description: This page lists changes to the error framework and deprecations.
 
-== Neo4j 2025.11
-**New:**
-[options="header", cols="<1m,<1"]
-|===
-| GQLSTATUS
-| Neo4j code
+= Changes and deprecations
```

Comment:
actually, now I'm starting thinking that we should place the information about the deprecated `failureReason` under the separate title.

````
== 2025.05

Starting from 2025.05, the `failureReason` in the JSON template of the query log, is deprecated and switched off by default.

Use the `errorInfo` entry instead.
This was introduced in 5.25.

To learn more about the JSON object `errorInfo` in the query log, visit the link:https://neo4j.com/docs/operations-manual/current/monitoring/logging/#gql-error-information[Operations manual -> Logging -> GQL error information].
````